examples/ex6p.cpp

```diff
@@ -169,31 +189,30 @@ int main(int argc, char *argv[])
       sout.precision(8);
    }
 
-   // 11. Set up an error estimator. Here we use the Zienkiewicz-Zhu estimator
+   // 12. Set up an error estimator. Here we use the Zienkiewicz-Zhu estimator
    //     with L2 projection in the smoothing step to better handle hanging
    //     nodes and parallel partitioning. We need to supply a space for the
    //     discontinuous flux (L2) and a space for the smoothed flux (H(div) is
    //     used here).
    L2_FECollection flux_fec(order, dim);
-   ParFiniteElementSpace flux_fes(&pmesh, &flux_fec, sdim);
+   ParFiniteElementSpace flux_fes(pmesh, &flux_fec, sdim);
    RT_FECollection smooth_flux_fec(order-1, dim);
-   ParFiniteElementSpace smooth_flux_fes(&pmesh, &smooth_flux_fec);
+   ParFiniteElementSpace smooth_flux_fes(pmesh, &smooth_flux_fec);
    // Another possible option for the smoothed flux space:
    // H1_FECollection smooth_flux_fec(order, dim);
-   // ParFiniteElementSpace smooth_flux_fes(&pmesh, &smooth_flux_fec, dim);
+   // ParFiniteElementSpace smooth_flux_fes(pmesh, &smooth_flux_fec, dim);
    L2ZienkiewiczZhuEstimator estimator(*integ, x, flux_fes, smooth_flux_fes);
 
-   // 12. A refiner selects and refines elements based on a refinement strategy.
+   // 13. A refiner selects and refines elements based on a refinement strategy.
    //     The strategy here is to refine elements with errors larger than a
    //     fraction of the maximum element error. Other strategies are possible.
    //     The refiner will call the given error estimator.
    ThresholdRefiner refiner(estimator);
    refiner.SetTotalErrorFraction(0.7);
 
-   // 13. The main AMR loop. In each iteration we solve the problem on the
+   // 14. The main AMR loop. In each iteration we solve the problem on the
    //     current mesh, visualize the solution, and refine the mesh.
-   const int max_dofs = 100000;
-   for (int it = 0; ; it++)
+   for (int it = 1; ; it++)
```

Comment:
I think it is more appropriate (and consistent with `ex6`) to call the first iteration "0" because it corresponds to the initial mesh and so no there's no AMR at that point.

Reply:
I forgot to change `ex6` to match the one-based iteration in ex6p, which I think is more natural for the user and also makes the checkpoint condition simpler (`it % 5 == 0` at the end of `ex6p`). Fixed in `ex6`.

Reply:
That's not the action I was expecting as a result of my comment.

If there is disagreement, I think it is appropriate to resolve it before further propagating the point of disagreement.

Reply:
@mlstowell, @tzanio, what is your preference regarding the above dispute?

Reply:
I agree that the two examples should be consistent, other than that I don't have a strong preference.

For what it's worth, Example 21 also starts from `0`: https://github.com/mfem/mfem/blob/master/examples/ex21.cpp#L192

Reply:
I don't think there is a disagreement, it's a negligible issue and both solutions are valid in my view. If there is a precedent in other examples let's change it accordingly.